crates/ruff_diagnostics/src/violation.rs

```diff
@@ -1,11 +1,21 @@
-use std::fmt::Debug;
+use std::fmt::{Debug, Display};
 
 pub enum AutofixKind {
     Sometimes,
     Always,
     None,
 }
 
+impl Display for AutofixKind {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            AutofixKind::Sometimes => write!(f, "Autofix is sometimes available."),
+            AutofixKind::Always => write!(f, "Autofix is always available."),
+            AutofixKind::None => write!(f, ""),
```

Comment:
Sorry for not having been clear about this in my original proposal. We should write `Autofix is not available.` in the case of `None` because this is a generic `Display` implementation.

Reply:
No problem, thank you for your review.